packages/ionio/src/Contract.ts

```diff
@@ -55,7 +56,8 @@ export class Contract implements ContractInterface {
     private artifact: Artifact,
     private constructorArgs: Argument[],
     private network: networks.Network,
-    private secp256k1ZKP: Secp256k1Interface
+    private secp256k1ZKP: Secp256k1Interface,
+    public internalPublicKey = H_POINT
```

Comment:
would this be backward compatible if other apps pull this version?

Reply:
Yes cause parameter is optional so former constructor statements are unchanged